h/accounts/views.py

```diff
@@ -574,6 +575,45 @@ def _update_subscription_data(request, subscription):
     request.session.flash(_('Changes saved!'), 'success')
 
 
+@view_config(attr='index', route_name='admin_users_index',
```

Comment:
If there's no real common code for these, you're welcome to decide that you think simple functions are easier and clearer than a class. It saves having to dereference the request from `self`.